requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Change in setup.py
 cython>=0.29.16; platform_python_implementation == 'CPython'
 numpy
-pillow
+pillow>=8.0.0
```

Comment:
What's the reason for this? It's preventing me from running tests on Python 3.7 on macOS, though I'm not exactly sure why...

Reply:
Okay, I seem to be able to work around it, so I guess it's just a quirk with using `python setup.py test`. Tests are running and I haven't had issues on any other platforms.

Reply:
Ah, I figured out what the problem was. I had `gym` installed, which requires `Pillow<=7.2.0`. So this probably shouldn't be merged until we address that.